maintainers/guide/creating-new-repos.md

```diff
@@ -1 +1,110 @@
+# Build GitHub Automation
 
+## Creating a new repository
+
+Using the `/build-repo` Slack Slash Command, the user will start the new repo creation process. The first step in this process involves checking the users `Slack ID` against an internal database to validate they have the correct permissions to create a new repo. Once that has been validated they will receive a new dialog window with the following information:
+
+| Variable         | Required |
+|------------------|----------|
+| Rubrik Team      | True     |
+| Repo Name        | True     |
+| Repo description | True     |
+
+The provided Rubrik team name will determine who is added as the repos `CODEOWNERS` along with who has write permissions to the repo. The only currently supported teams are:
+
+* OCTO
+* Engineering
+
+### OCTO Specific Settings
+
+* `CODEOWNERS` file will include `*       @rubrikinc/octo`
+* `OCTO` added as a Team on the repo with `write` permissions
+
+### Engineering Specific Settings
+
+*  New GitHub team will be created that matches the name of the repository
+* `CODEOWNERS` file will include `*       @rubrikinc/{repo-name}`
+* `{repo-creators-github-username}` added to the new team
+
+### Repository Standards
+
+* Naming convention: `{repo-name}` (i.e all lowercase with dashes)
+* License: `MIT`
+* Default branch: `master`
+* Branch protection rules: `enabled` (master)
+  * Require pull request reviews before merging: `enabled`
+    * Required approving reviews: `1`
+    * Dismiss stale pull request approvals when new commits are pushed: `enabled`
+    * Require review from Code Owners: `enabled`
+  * Require status checks to pass before merging: `enabled` (Note: User will still need to select which status checks are required. By default, none are selected as a requirement.)
+    * Require branches to be up to date before merging: `enabled`
+  * Require signed commits: `enabled`
+  * Include administrators: `disabled`
+* Files created
+  * top level `README.md`
+  * `CODE_OF_CONDUCT.md`
+  * `CONTRIBUTING.md`
+  * `.github/CODEOWNERS`
+  * `.github/pull_request_template.md`
+  * `.github/ISSUE_TEMPLATE/bug-report.md`
+  * `.github/ISSUE_TEMPLATE/`
+  * `.github/enhancement_request.md`
+* Labels
+
+## Adding a new user to the `rubrikinc` GitHub organization
+
```

Comment:
The `/build-repo` command was mentioned above - should we not also provide the slack command for adding users/managing permissions?

Reply:
Absolutely. We just have to create them first :) 

`/build-repo` was a little easier since that was based off an existing workflow we used for the OCTO (aka Ranger team. 

Once they are in place I'll update the guide here. I'll probably also break each section out into their own markdown file as well.